src/script/user/UserRepository.ts

```diff
@@ -84,14 +84,14 @@ export interface UserUpdate {
 export class UserRepository {
   private readonly asset_service: AssetService;
   private readonly client_repository: ClientRepository;
-  private readonly connected_users: ko.PureComputed<User[]>;
-  private readonly isTeam: ko.Observable<boolean>;
+  readonly connected_users: ko.PureComputed<User[]>;
+  isTeam: ko.Observable<boolean> | ko.PureComputed<boolean>;
   private readonly logger: Logger;
   private readonly propertyRepository: PropertiesRepository;
   private readonly selfService: SelfService;
-  private readonly teamMembers: ko.ObservableArray<User>;
+  teamMembers: ko.ObservableArray<User> | ko.PureComputed<User[]>;
```

Comment:
How come that it can be of two different types?

Reply:
Because they are [set in `TeamRepository.ts#L114`](https://github.com/wireapp/wire-webapp/blob/c05df99acd4429618d8ab99b8dbaaae3c3cc7975/src/script/team/TeamRepository.ts#L114).